menus/menu-main.sh

```diff
@@ -6,6 +6,9 @@ then
   echo
   exit 0
 fi
+
+APPSETTINGSTYPE='NA'
```

Comment:
Apparently I need to look at everything in `/inc`, I didn't even know that script was there! haha

That sounds good, I'll push that change.